evap/results/tests/test_views.py

```diff
@@ -699,3 +701,35 @@ def test_archived_results(self):
         self.app.get(url, user='manager', status=200)
         self.app.get(url, user='reviewer', status=403)
         self.app.get(url, user='student_external', status=403)
+
+
+class TestTextAnswerExportView(WebTest):
+
+    @classmethod
+    def setUpTestData(cls):
+        baker.make(UserProfile, username="reviewer", groups=[Group.objects.get(name="Reviewer")])
+        evaluation = baker.make(Evaluation)
+
+        baker.make(TextAnswer, _quantity=3, __contribution__evaluation=evaluation)
```

Comment:
The two leading underscores in `__contribution__evaluation` are smelly, I quickly checked and it looks like baker does not use that argument. (This is really strange as it also doesn't give an error, I would expect that it somehow gives an error if you pass it some kwarg that it does not understand -- will look into that a bit more later).

Do you need these TextAnswers? From what I see, you patch / mock the export method in both tests, so they shouldn't be necessary - is that correct?